central/notifiers/email/email.go

```diff
@@ -552,6 +546,16 @@ func (e *email) getPassword() (string, error) {
 		return e.creds, nil
 	}
 
+	if e.notifier.GetNotifierSecret() == "" {
+		return "", errors.Errorf("encrypted notifier credentials for notifier '%s' empty", e.notifier.GetName())
+	}
+	if e.cryptoCodec == nil {
```

Comment:
same as above

Reply:
Removed the empty check for key, but I think we can keep this one in-case someone called the NewEmail and similar funs with a nil value for codec.